docs/guides/admin/docs/modules/ltimodule.md

```diff
@@ -14,48 +14,51 @@ More information about the LTI specifications is available at
+
+```xml
     <!-- 2-legged OAuth is used by trusted 3rd party applications, including LTI. -->
     <!-- Uncomment the line below to support LTI or other OAuth clients.          -->
     <ref bean="oauthProtectedResourceFilter" />
-````
-
-* Replace CONSUMER_KEY and CONSUMER_SECRET with LTI the key and secret values that you will use in your LMS:
-````
-    <!-- ####################### -->
-    <!-- # OAuth (LTI) Support # -->
-    <!-- ####################### -->
-
-    <!-- This is required for LTI. If you are using LTI and have enabled the oauthProtectedResourceFilter  -->
-    <!-- in the list of authenticationFilters above, set custom values for CONSUMERKEY and CONSUMERSECRET. -->
-
-    <bean name="oAuthConsumerDetailsService" class="org.opencastproject.kernel.security.OAuthSingleConsumerDetailsService">
-    <constructor-arg index="0" ref="userDetailsService" />
-    <constructor-arg index="1" value="CONSUMERKEY" />
-    <constructor-arg index="2" value="CONSUMERSECRET" />
-    <constructor-arg index="3" value="constructorName" />
-    </bean>
-````
-
-* To give LMS users the same username in Opencast as the LMS username, uncomment the constructor arguments
-below and update CONSUMERKEY to the same key used above:
-
-````
-    <!-- Uncomment to trust usernames from the LTI consumer identified by CONSUMERKEY.           -->
-    <!-- Users from untrusted systems will be prefixed with "lti:" and the consumer domain name. -->
-
-    <constructor-arg index="1" ref="securityService" />
-    <constructor-arg index="2">
-      <list>
-        <value>CONSUMERKEY</value>
-      </list>
-    </constructor-arg>
-````
+```
+
+To configure OAuth consumers (e.g. a LMS), edit
+`OPENCAST/etc/org.opencastproject.kernel.security.OAuthConsumerDetailsService.cfg` and replace CONSUMERNAME,
+CONSUMERKEY, and CONSUMERSECRET with the values you will use in your LMS:
+
+```properties
+oauth.consumer.name.1=CONSUMERNAME
+oauth.consumer.key.1=CONSUMERKEY
+oauth.consumer.secret.1=CONSUMERSECRET
+```
+
+### Configure LTI (optional)
+
+To give LMS users the same username in Opencast as the LMS username, edit
+`etc/org.opencastproject.kernel.security.LtiLaunchAuthenticationHandler.cfg` and add the configured OAuth consumer key
+to the list of highly trusted keys.
+
+```properties
+lti.oauth.highly_trusted_consumer_key.1=CONSUMERKEY
+```
+
+Use can exempt specific users even if a highly trusted consumer is used by configuring a blacklist. Additionally, there
+are settings for excluding the system administrator as well as the digest user (enabled by default).
+
+```properties
+lti.allow_system_administrator=false
+lti.allow_digest_user=false
+lti.blacklist.user.1=myAdminUser
+```
+
+> **Notice:** Marking a consumer key as highly trusted can be a security risk! If the usernames of sensitive Opencast
+> users are not blacklisted, the LMS administrator could create LMS users with the same username and use LTI to grant
```

Comment:
Can we list these usernames?  I'd think it would only be the two, and those are not likely to change, correct?

Reply:
By default yes, but there might be many more (e.g. custom admin or API users). However, I can list the default usernames.

Reply:
Done